features/karaf/src/main/feature/feature.xml

```diff
@@ -28,6 +28,7 @@
         <bundle prerequisite="true">mvn:org.apache.karaf.shell/org.apache.karaf.shell.core/${dep.karaf.version}</bundle>
         <bundle prerequisite="true">mvn:org.apache.karaf.wrapper/org.apache.karaf.wrapper.core/${dep.karaf.version}</bundle>
         <bundle>mvn:org.openhab.core/org.openhab.core.karaf/${project.version}</bundle>
+        <feature>openhab-runtime-certificate</feature>
```

Comment:
I'd prefer to move this line up to the other features, so that we first have all features and then the bundles listed.